webserver/views/api/v1/core.py

```diff
@@ -203,6 +220,8 @@ def get_many_lowlevel():
       Takes the form `mbid[:offset];mbid[:offset]`. Offsets are optional, and should
       be >= 0
 
+      You can specify up to :py:const:`~webserver.views.api.v1.core.MAX_ITEMS_PER_BULK_REQUEST` MBIDs in a request.
```

Comment:
noice!

Reply:
this is kind of OK. It links to the definition elsewhere in the docs, but I would have preferred to make it put the actual number in the docstring.